app/views/content/landing/campus-advisers/_header.html.erb

```diff
@@ -1,6 +1,5 @@
-<%= render Content::CampaignHeroComponent.new(
-  title: ["Sign up for an", "adviser"],
+<%= render Content::LandingHeroComponent.new(
+  title: "Sign up for an adviser",
```

Comment:
The descender on the `g` in `Sign` is slightly cut-off on my phone (iPhone 11)

https://get-into-teaching-app-review-3942.test.teacherservices.cloud/landing/campus-advisers

Reply:
Same on `/landing/campus-mailing-list`

Reply:
have adjusted the line height slightly on this